docs/configuration/snippets/_configuration_properties.properties

```diff
@@ -10,6 +10,7 @@ springwolf.docket.info.contact.name=springwolf
 springwolf.docket.info.contact.email=example@example.com
 springwolf.docket.info.contact.url=https://github.com/springwolf/springwolf-core
 springwolf.docket.info.license.name=Apache License 2.0
+springwolf.docket.info.x-internal-id=xyz-123
 
 springwolf.docket.servers.kafka.protocol=kafka
-springwolf.docket.servers.kafka.url=${spring.kafka.bootstrap-servers}
```

Comment:
IDE added additional, traling <lf> to last line of properties file. I think, that's ok...